spec/requests/api/generic_events_controller_create_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'rails_helper'
 
+# rubocop:disable RSpec/MultipleMemoizedHelpers
```

Comment:
We're disabling this a lot - do we ever want it? Is there a way to disable it in one place for all specs?